### PR TITLE
Bugfix/303 spec region is not being used

### PR DESCRIPTION
**Pull request by g-iannelli**
- updated infrastructure phase terraform templates
- updated kubernetes phase terraform templates
- remove AWS_DEFAULT_REGION env var validation when use resource of kind EKSCluster

### CODE REVIEW

File: configs/provisioners/cluster/eks/data.tf [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    module.fury
+  ]
+}
+
+data "aws_eks_cluster_auth" "fury" {
+  name = var.cluster_name
+  depends_on = [
+    module.fury
+  ]
+}
```
Comment (Al-Pragliola) [inline configs/provisioners/cluster/eks/data.tf:13]: missing newline at end of file, also add license at start

**Review by Al-Pragliola — CHANGES_REQUESTED** [review]
Code-wise all seems good, but you should fix the CI and remove from the code every reference to AWS_DEFAULT_REGION (eg. .drone.yml, e2e tests, ... )

**Review by Al-Pragliola — APPROVED** [review]
LGTM

**Review by nutellinoit — APPROVED** [review]
LGTM 🚀

**Pull request merged**